### DISCUSSION TITLE
How can i make use o Custom Controls?

### DISCUSSION THREAD

**Discussion by felipe-labbits** [jhomlala/betterplayer#503] [Q&A]
I would like to customize all controls in the player,... i tried to use "customControlsBuilder" but it never render, i really don't get it.

Which is the best way to customize controls?

**Comment by jhomlala** [comment]
Better Player offers set of configuration options for default controls: https://github.com/jhomlala/betterplayer/blob/master/example/lib/pages/controls_configuration_page.dart

This is how you can build custom controls: https://github.com/jhomlala/betterplayer/tree/master/example/lib/pages/custom_controls

**Reply by darter0** [reply]
when writing custom controls they work fine, but they are always visible and stuck on screen,
is that related to onControlsVisibilityChanged??
how to properly make the custom controls disappear like default conrtolls?
more details:
https://stackoverflow.com/questions/79055953/flutter-better-player-custom-controls-problem
